backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceCandBLCreateInvoices.java

```diff
@@ -433,6 +433,10 @@ private I_C_Invoice createInvoice(
 
 			invoice.setPaymentRule(invoiceHeader.getPaymentRule());
 			invoice.setM_SectionCode_ID(SectionCodeId.toRepoId(invoiceHeader.getM_SectionCode_ID()));
+
+			invoice.setInvoiceAdditionalText(invoiceHeader.getInvoiceAdditionalText());
+			invoice.setIsNotShowOriginCountry(invoiceHeader.isNotShowOriginCountry());
```

Comment:
Instead of here, you should be populating this field in: `de.metas.invoice.service.impl.AbstractInvoiceBL#createInvoiceFromOrder`. That way, there's no need for it to be in the InvoiceHeader.

Reply:
To copy the new fields from Invoice candidate to the invoice

Reply:
Besides `AbstractInvoiceBL#createInvoiceFromOrder` (which I didn't dig into now), i would like those properties to enter the invoice via C_Invoice_Candidate and not directly coming form order 
(because we have many sources for invoice candidates and in future they might also want to control these two properties)